### PR TITLE
chapter 11 tech review fixes

### PR DESCRIPTION
**Pull request by louissalin**

### CODE REVIEW

File: chapter-10/end/States/Splash/SplashState.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class SplashState : BaseGameState
     {
```
Comment (jcapellman) [inline chapter-10/end/States/Splash/SplashState.cs:10]: Just confirming you meant to modify Chapter-10's end state?

File: chapter-11/end/Engine/Objects/BaseGameObject.cs [hunk 2]
```diff
@@ -12,6 +12,8 @@ public class BaseGameObject
         protected Texture2D _boundingBoxTexture;
 
         protected Vector2 _position = Vector2.One;
+        protected float _angle;
```
Comment (jcapellman) [inline chapter-11/end/Engine/Objects/BaseGameObject.cs:15]: Nice refactoring moving this to the base class

**Review by jcapellman — APPROVED** [review]

File: chapter-10/end/States/Splash/SplashState.cs (hunk 1 above)
Comment (louissalin) [reply chapter-10/end/States/Splash/SplashState.cs:10]: yes. The content pipeline was restructured in this chapter so the splashscreen moved to Images/ in the end state.

File: chapter-11/end/Engine/Objects/BaseGameObject.cs (hunk 2 above)
Comment (louissalin) [reply chapter-11/end/Engine/Objects/BaseGameObject.cs:15]: I wish I could take credit. The tech reviewer caught this.

File: chapter-10/end/States/Splash/SplashState.cs (hunk 1 above)
Comment (jcapellman) [reply chapter-10/end/States/Splash/SplashState.cs:10]: Ok cool - just wanted to be sure 👍

**Pull request merged**